packages/engine-test-utils/src/__tests__/engine-server/markdown/remarkUtils.spec.ts

```diff
@@ -5,27 +5,61 @@ import {
   MDUtilsV4,
   RemarkUtils,
   LinkUtils,
+  DendronASTTypes,
 } from "@dendronhq/engine-server";
 import _ from "lodash";
 import { DVault, runEngineTestV5, testWithEngine } from "../../../engine";
 import { checkString } from "../../../utils";
 
-describe("utils", () => {
-  describe("findHeaders", async () => {
+describe("RemarkUtils and LinkUtils", () => {
+  describe("findAnchors", async () => {
     test("one header", async () => {
       await runEngineTestV5(
         async ({ engine }) => {
           const body = engine.notes["foo"].body;
-          const out = RemarkUtils.findHeaders(body);
+          const out = RemarkUtils.findAnchors(body);
           expect(out).toMatchSnapshot("bond");
           expect(_.size(out)).toEqual(1);
           expect(out[0].depth).toEqual(1);
+          expect(out[0].type).toEqual(DendronASTTypes.HEADING);
         },
         {
           preSetupHook: async ({ vaults, wsRoot }) => {
             await NoteTestUtilsV4.createNote({
               fname: "foo",
-              body: "# h1",
+              body: [
+                "# h1",
+                "",
+                "Repellendus possimus voluptates tempora quia.",
+              ].join("\n"),
+              vault: vaults[0],
+              wsRoot,
+            });
+          },
+          expect,
+        }
+      );
+    });
+    test("one block anchor", async () => {
+      await runEngineTestV5(
+        async ({ engine }) => {
+          const body = engine.notes["foo"].body;
+          debugger;
+          const out = RemarkUtils.findAnchors(body);
+          expect(out).toMatchSnapshot("bond");
```

Comment:
"bond" is a debug value to help quickly find a given snapshot. you can take a snapshot without parameters. if you see other snapshots using this value, please remove them